packages/react-native-gradle-plugin/src/test/kotlin/com/facebook/react/tasks/internal/PrepareBoostTaskTest.kt

```diff
@@ -37,13 +37,13 @@ class PrepareBoostTaskTest {
           it.boostVersion.set("1.0.0")
           it.outputDir.set(output)
         }
-    File(project.projectDir, "src/main/jni/third-party/boost/Android.mk").apply {
+    File(project.projectDir, "src/main/jni/third-party/boost/CMakeLists.txt").apply {
```

Comment:
Nit: the name of this test is `prepareBoostTask_copiesMakefile`. Can you update it to `prepareBoostTask_copiesCMakefile` (same for all the tests)